common/convert.py

```diff
@@ -13,19 +13,17 @@ def timedelta_to_string(data: timedelta) -> str:
     return "{:02d}:{:02d}".format((hours + (days * 24)) if days > 0 else hours, minutes)
 
 
-def convert_string_work_hours_field_to_hour_and_minutes(
-    data: str, exception: Union[RestValidationError, DjangoValidationError]
-) -> Tuple[str, str]:
+def convert_string_work_hours_field_to_hour_and_minutes(data: str) -> Tuple[str, str]:
     # parameter `exception` is only workaround until we get rid of rest_framework views.
```

Comment:
please remove comment - it is no longer valid ;)

Reply:
:+1: done